lib/streamlit/elements/vega_charts.py

```diff
@@ -607,9 +598,6 @@ def line_chart(
         the chart's Altair spec. As a result this is easier to use for many
         "just plot this" scenarios, while being less customizable.
 
-        If ``st.line_chart`` does not guess the data specification
```

Comment:
Removing these lines because they aren't really accurate anymore today given that you can use `x` and `y` to specify which columns to use.